.github/workflows/docs-update.yml

```diff
@@ -0,0 +1,36 @@
+name: Update OpenTelemetry Website Docs
+
+on:
+  # triggers only on a manual dispatch
+  workflow_dispatch:
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2.3.4
+      - name: make-pr
+        env:
+          API_TOKEN_GITHUB: ${{secrets.DOC_UPDATE_TOKEN}}
```

Comment:
I assume once this PR is merged we will need to add the token, right? Let's do it after it is merged and do a test run of the workflow to see how it works.

Reply:
@austinlparker can you please add the token and let's see how the docs-update works.